CHANGELOG.md

```diff
@@ -6,7 +6,33 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- `prefix_list` function in `utils.util.py`
+- `KBinsDiscretizer` preprocessing model
+- `PowerTransformer` preprocessing model
+- `SplineTransformer` preprocessing model
+- `TargetEncoder` preprocessing model
+- `QuantileTransformer` preprocessing model
+- `RobustScaler` preprocessing model
+- `PolynomialFeatures` preprocessing model
+- `OrdinalEncoder` preprocessing model
+- `Normalizer` preprocessing model
+- `MaxAbsScaler` preprocessing model
+- `MultiLabelBinarizer` preprocessing model
+- `KernelCenterer` preprocessing model
+- `FunctionTransformer` preprocessing model
+- `Binarizer` preprocessing model
+- `tests/test_preprocessings/util.py`
```

Comment:
remove this line

Reply:
fixed in [3d6eacb](https://github.com/openscilab/pymilo/pull/104/commits/3d6eacb5a617611e808f9fb2aa07f9e3c8d8c701).